nomenclature/region_mapping_models.py

```diff
@@ -24,35 +29,70 @@
 
 here = Path(__file__).parent.absolute()
 
-Sequence = Union[List[str], Tuple[str], Set[str]]
-
-
-def get_relative_path(path: Path):
-    # Get the relative version of `path` relative to `path_relative_to`
-    # In case path does not contain path_relative_to it is returned unchanged
-    return path.relative_to(Path.cwd()) if path.is_absolute() else path
 
+class NativeRegion(BaseModel):
+    """Defines a model native region.
 
-class RegionNameCollisionError(PydanticValueError):
-    code = "region_name_collision"
-    msg_template = "Name collision in {location} for {duplicates} in {file}"
+    Can optionally have a renaming attribute which is applied in the region processing.
 
+    Attributes
+    ----------
+    name : str
+        Name of the model native region.
+    rename: Optional[str]
+        Optional second name that the region will be renamed to.
+    """
 
-class NativeRegion(BaseModel):
     name: str
     rename: Optional[str]
 
     @property
-    def target_native_region(self):
+    def target_native_region(self) -> str:
+        """Returns the resulting name, i.e. either rename or, if not given, name.
+
+        Returns
+        -------
+        str
+            Resulting name
```

Comment:
```suggestion
            Resulting name.
```
I guess convention in this module includes a dot.